client/layout/guided-tours/tours/jetpack-site-accelerator-tour/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import meta from './meta';
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Quit,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+export const JetpackSiteAcceleratorTour = makeTour(
+	<Tour { ...meta }>
+		<Step
+			name="init"
+			target=".jetpack-site-accelerator-settings .form-toggle__switch"
+			arrow="top-left"
+			placement="below"
+			style={ {
+				animationDelay: '0.7s',
+				zIndex: 1,
+			} }
+		>
+			{ ( { translate } ) => (
+				<Fragment>
+					<p>{ translate( 'Activate this toggle to enable Site Accelerator' ) }</p>
+					<ButtonRow>
+						<Continue
+							target=".jetpack-site-accelerator-settings .form-toggle__switch"
+							step="finish"
+							click
+							hidden
+						/>
+						<SiteLink href="/plans/my-plan/:site">
+							{ translate( 'Return to the checklist' ) }
+						</SiteLink>
+					</ButtonRow>
+				</Fragment>
+			) }
+		</Step>
+
+		<Step name="finish" placement="right">
+			{ ( { translate } ) => (
+				<Fragment>
+					<h1 className="tours__title">
+						<span className="tours__completed-icon-wrapper">
+							<Gridicon icon="checkmark" className="tours__completed-icon" />
+						</span>
+						{ translate( 'Excellent, you’re done!' ) }
+					</h1>
+					<p>
+						{ translate(
+							'Site Accelerator has been enabled. Would you like to continue setting up performance features for your site?'
```

Comment:
Site Accelerator is active on your site. Next up: faster video loading. Ready to move on?

(If you don't want to use variations for all these "next?" prompts, that's fine. Let's just change tham all to get rid of the passive voice and use the "Feature X is active" phrasing.)